test/dotcom_web/controllers/trip_plan_controller_test.exs

```diff
@@ -813,7 +813,7 @@ defmodule DotcomWeb.TripPlanControllerTest do
     end
 
     test "doesn't set custom_route? flag for regular routes", %{itineraries: itineraries} do
-      expect(MBTA.Api.Mock, :get_json, fn "/routes/" <> _, _ ->
+      stub(MBTA.Api.Mock, :get_json, fn "/routes/" <> _, _ ->
```

Comment:
changed because may or may not get called depending on the itinerary (needs improved factory to create better base itineraries)